Loop/Views/SettingsView.swift

```diff
@@ -194,6 +194,7 @@ public struct SettingsView_Previews: PreviewProvider {
                                           cgmManagerSettingsViewModel: DeviceViewModel(),
                                           therapySettings: TherapySettings(),
                                           supportedBasalRates: nil,
+                                          supportedBolusVolumes: nil,
```

Comment:
Same comment as on #158 about getting this value from the pumpManager or view model